ydb/services/ydb/ut/ya.make

```diff
@@ -38,7 +38,7 @@ PEERDIR(
     library/cpp/regex/pcre
     library/cpp/svnversion
     ydb/core/kqp/ut/common
-    ydb/core/testlib/default
+    ydb/core/testlib/pg
```

Comment:
Is it intentionally?

Reply:
Yes, testlib/default has dummy implementation for pg types interfaces and catalog